doc/api/stream.md

```diff
@@ -325,6 +349,9 @@ implementations that implement the `writable.\_writev()` method can perform
 buffered writes in a more optimized manner.
 
 ##### writable.end([chunk][, encoding][, callback])
+<!-- YAML
+added: v0.9.9
+-->
```

Comment:
Yeah, but that `0.9.4` that method doesn't accept a callback

https://github.com/nodejs/node/commit/a9c4a20331d123dc38dcc878af335d88e1933562